src/manage/install_command.py

```diff
@@ -329,25 +329,43 @@ def update_all_shortcuts(cmd, path_warning=True):
     for k, (_, cleanup) in SHORTCUT_HANDLERS.items():
         cleanup(cmd, shortcut_written.get(k, []))
 
-    if path_warning and cmd.global_dir and cmd.global_dir.is_dir() and any(cmd.global_dir.glob("*.exe")):
+
+def print_cli_shortcuts(cmd):
+    if cmd.global_dir and cmd.global_dir.is_dir() and any(cmd.global_dir.glob("*.exe")):
         try:
             if not any(cmd.global_dir.match(p) for p in os.getenv("PATH", "").split(os.pathsep) if p):
                 LOGGER.info("")
                 LOGGER.info("!B!Global shortcuts directory is not on PATH. " +
-                            "Add it for easy access to global Python commands.!W!")
+                            "Add it for easy access to global Python aliases.!W!")
                 LOGGER.info("!B!Directory to add: !Y!%s!W!", cmd.global_dir)
                 LOGGER.info("")
+                return
         except Exception:
             LOGGER.debug("Failed to display PATH warning", exc_info=True)
+            return
 
-
-def print_cli_shortcuts(cmd):
     installs = cmd.get_installs()
-    seen = set()
+    tags = getattr(cmd, "tags", None)
+    seen = set("python.exe".casefold())
+    verbose = LOGGER.would_log_to_console(VERBOSE)
     for i in installs:
-        aliases = sorted(a["name"] for a in i["alias"] if a["name"].casefold() not in seen)
-        seen.update(n.casefold() for n in aliases)
-        if not install_matches_any(i, cmd.tags):
+        # We only show windowed aliases if -v is enabled. But we log them as
```

Comment:
It's actually the `-64` aliases that I think are least useful to the average user. I'm more inclined to remove them than the windowed ones, if it's one or the other.

Hmm, if the user has both `-64` and `-32` builds present which one will `python3.13` run? And does the printing process here take *that* into account?

Reply:
Ah, good point. Those suffixes already get some special logic, so it doesn't hurt to add a bit more.

The default platform (another config setting, defaults to `-64` right now but will choose `-arm64` one day in the future on that arch) should determine which runtime gets the `python3.13` alias.